### PR TITLE
WI00764279 -- NCN Diagrams Move wtg.blazor.diagrams to nuget.org

### PR DESCRIPTION
**Pull request by vinnie-fialok**
- Updated release workflow to additionally deploy to nuget
- Updated README to replace Z.Blazor.Diagrams with WTG.Z.Blazor.Diagrams

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,20 +10,22 @@ env:
...
+    - name: Checkout
+      uses: actions/checkout@v3
+
     - name: Setup dotnet
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: |
             6.0.x
             3.1.x
-
```
Comment (LeonamAnjos) [inline .github/workflows/release.yml:25]: Can we remove dotnet-version 3.1.x? Looks like we are deploying only 6.0.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -9,20 +10,22 @@ env:
...
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: |
             6.0.x
             3.1.x
-      
+
       # Finds the latest release and increases the version
     - name: Get next version
       uses: reecetech/version-increment@2023.9.3
```
Comment (LeonamAnjos) [inline .github/workflows/release.yml:31]: How can we ensure that we start with version 4.0.18 in nuget.org?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -59,3 +62,8 @@ jobs:
       with:
         files: ${{ env.PACKAGE_PATH }}
         tag_name: ${{ steps.version.outputs.version }}
+
+    - name: Push NuGet Package to NuGet Gallery
+      run: |
+        nuget setapikey ${{ secrets.NUGET_API_KEY }}
+        nuget push ${{ env.PACKAGE_PATH }} -Source https://api.nuget.org/v3/index.json
```
Comment (LeonamAnjos) [inline .github/workflows/release.yml:69]:
```suggestion
        nuget push ${{ env.PACKAGE_PATH }} -Source https://api.nuget.org/v3/index.json

```

File: README.md [hunk 4]
```diff
@@ -2,13 +2,11 @@
 
 ![](ZBD.png)
 
-Z.Blazor.Diagrams is a fully customizable and extensible all-purpose diagrams library for Blazor (both Server Side and WASM). It was first inspired by the popular React library [react-diagrams](https://github.com/projectstorm/react-diagrams), but then evolved into something much bigger. ZBD can be used to make advanced diagrams with a custom design. Even the behavior of the library is "hackable" and can be changed to suit your needs. 
+WTG.Z.Blazor.Diagrams is a fully customizable and extensible all-purpose diagrams library for Blazor (both Server Side and WASM). It was first inspired by the popular React library [react-diagrams](https://github.com/projectstorm/react-diagrams), but then evolved into something much bigger. ZBD can be used to make advanced diagrams with a custom design. Even the behavior of the library is "hackable" and can be changed to suit your needs.
```
Comment (LeonamAnjos) [inline README.md:5]: Hum... not sure if it is valid for `WTG.Z.Blazor.Diagrams`, as it is a fork from `Z.Blazor.Diagrams`. Maybe we should mention that way. It is a `Z.Blazor.Diagrams` with new features and bug fixes first targeting WiseTech Global products.
@Heathermcx , wdyt?

File: .github/workflows/release.yml (hunk 2 above)
Comment (vinnie-fialok) [reply .github/workflows/release.yml:31]: For this, we'd likely need a seperate github action where we can specify the version with a tag. However, I have tested that it does pick out the next version as 4.0.18, and merging to master **should** result in version 4.0.18.

Ref: https://github.com/WiseTechGlobal/Blazor.Diagrams/actions/runs/9801236216/job/27064259461

Specifically in the **Get next version** step we have `The new version is 4.0.18-pre.9c30c92`

File: README.md (hunk 4 above)
Comment (vinnie-fialok) [reply README.md]: Have updated this to be more accurate regarding what WTG.Z.Blazor.Diagrams is

File: .github/workflows/release.yml (hunk 2 above)
Comment (LeonamAnjos) [reply .github/workflows/release.yml:29]: Great! [Here](https://github.com/reecetech/version-increment/blob/main/version-lookup.sh#L35) sorts the tags we have on GitHub and picks the last one.

**Review by LeonamAnjos — APPROVED** [review]
LGTM

**Review by Heathermcx — APPROVED** [review]

**Pull request merged**